tests/validate-req-traceability.sh

```
@@ -0,0 +1,144 @@
+    exit 2
+fi
+
+# --- Parse manifest ---
+declare -A MANIFEST_ENTRIES
+declare -A EXEMPTIONS
+UNKNOWN_REFS=()
+DUPLICATES=()
+MANIFEST_COUNT=0
+
+header_seen=0
+while IFS=$'\t' read -r req_id coverage reference notes || [[ -n "$req_id" ]]; do
```

Shellcheck (0.11.0)

[warning] reference appears unused. Verify use (or export if used externally).

(SC2034)